plutus-use-cases/test/Spec/crowdfundingEmulatorTestOutput.txt

```diff
@@ -186,11 +178,7 @@ Slot 20: W[1]: Balancing an unbalanced transaction:
                        , "U}#\192\165\&3\180\210\149\172-\193Kx:~\252);\194>\222\136\166\254\253 =" )
                        ( 77ab184b7537cd4b1dc3730f6a8a76a3d3aad1642fae9d769aa5dae40be38b51
                        , "\128\164\244[V\184\141\DC19\218#\188L<u\236m2\148<\b\DEL%\v\134\EM<\167" )
-                     attached scripts:
-                       ( c3387c08305c03856c391b6c1b1bc872331a3a238fbaf72051142264
-                       , Plutus V2 )}
-                 Requires signatures:
-                   a2c20c77887ace1cd986193e4e75babd8993cfd56995cd5cfce609c2
+                     redeemers:}
```

Comment:
is it correct that `redeemers` is empty? shouldn't it contain the scripts and script data?